Makefile

```diff
@@ -135,9 +135,20 @@ install: all
 	    install -d $(INSTALL_PREFIX)/share/man/man1; \
 	    install -pm 644 -t $(INSTALL_PREFIX)/share/man/man1 man/*.1; \
 	fi
-#       misc "documentation"
+#       license and readme
 	install -d $(DOC)
 	install -pm 644 -t $(DOC) LICENSE README.rst
+#	html files if they were built
+	if (ls -A doc | grep -q .); then \
```

Comment:
[Don't parse `ls`](http://mywiki.wooledge.org/ParsingLs). You want to see if there are any HTML files in there, right? I think something [based on `find(1)`](https://stackoverflow.com/a/15663760) would be better.

Reply:
Actually, we know the filenames, so we don't need `find(1)`. I think something like `if [ -f doc/index.html ]` would do the trick.